sys/random/Kconfig

```diff
@@ -17,7 +17,7 @@ choice RANDOM_IMPLEMENTATION
     bool "PRNG Implementation"
     depends on TEST_KCONFIG
     default MODULE_PRNG_HWRNG if HAS_PERIPH_HWRNG
-    default MODULE_PRNG_TINYMT32
+    default MODULE_PRNG_TINYMT32 if !HAS_PERIPH_HWRNG
```

Comment:
Kconfig already has the described behaviour. I don't think this is needed, the first default (top to bottom) that can be applied will be applied.